File: FE-RESTAURANT/src/interfaces/CategoryResponseDTO.ts
import { CategoryStatus } from './enums/CategoryStatus';

export interface CategoryResponseDTO {
  id: string;
  name: string;
  description: string;
  status: CategoryStatus;
  createdBy: string;
  createdAt: string;
  updatedAt: string;
  inUse?: boolean;
} 

File: RESTAURANT/src/main/java/com/poly/restaurant/dtos/CategoryResponseDTO.java
package com.poly.restaurant.dtos;

import com.poly.restaurant.entities.enums.CategoryStatus;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CategoryResponseDTO {
    private Long id;
    private String name;
    private String description;
    private CategoryStatus status;
    private Long branchId;
    private String branchName;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
    private Boolean inUse;
}